src/graphql/assets.ts

```diff
@@ -44,19 +10,29 @@ const AssetContentType = {
  * @param asset CMA Asset object containing the update
  * @param locale locale code
  */
-export function updateAsset(
-  data: Entity & { sys: SysProps },
+export function updateAsset<T extends (Entity & { sys: SysProps }) | EntryProps>(
+  data: T,
```

Comment:
```suggestion
  dataFromPreviewApp: T,
```

Reply:
I find the naming here a bit more clear, it's also in line with the other naming in the other files

Reply:
yes you're right, oversaw that it has still the old names 🙈